### DISCUSSION TITLE
GraphJin, A New Kind of ORM

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#1672] [Show and tell]
@ganigeorgiev GraphJin 2.0 is out.  Apparently you can write GraphQL and it will translate it into DB calls.

https://github.com/dosco/graphjin

**Comment by ganigeorgiev** [comment]
I'm aware of graphjin but I don't see how this is related to PocketBase.

GraphQL API at least for v1 is not planned (you can check https://github.com/pocketbase/pocketbase/discussions/9).
